index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/styles.css">
+    <title>Fabricio La Torre</title>
+</head>
+<body>
+    <header class="main-header">
+        <div class="container">
+            <h1 class="mh-logo">
+              <a href="#"><img alt="kiwilogo" src="images/kiwilogo.svg"></a>
+                
+            </h1>
+            <nav class="main-nav">
+               
+                <ul class="main-nav-list">
+                  
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                    <li>
+                        <a href="#"><img alt="maillogo" src="images/mail.svg"></a>
+                    </li>
+                </ul>
+            </nav>
+           
+        </div>
+
+    </header>
+
+
+    <section class="body-container">
+   
+
+      <ul class="intro">
+        <li>
+          <h1 class="name">
+              Hello! I'm Fabricio La Torre.
+          </h1>
+          <h3>
+              I'm a Web Developer 
+          </h3>
+          <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nobis doloremque ea sequi ullam dolor animi, cumque ab voluptate iusto sit.
+          </p>
+        </li>
+    
+      </ul>
```

Comment:
- [ ] It is important to acknowledge what you have done so far. Great job 👏. In this section, it looks like you misunderstood what the reviewer was trying to tell you. If you try to view your work before adding these changes 👇 
![image](https://user-images.githubusercontent.com/90222110/176766355-181872e4-accc-481f-ad66-feff330dc411.png)
This was correct and you didn't have to change it. What he wanted to say was to add the social media icons and also group them in list. Look below the text paragraph in Figma design. Picture below 👇 
![image](https://user-images.githubusercontent.com/90222110/176767360-f377ea96-8fe6-4a62-97fd-14ce3b427efd.png)
![image](https://user-images.githubusercontent.com/90222110/176768736-a5292f44-c57b-4504-9f09-03ae89560006.png)

These are the ones you have to add using list tags and they should be links. Let me give you the format which you can refer to while you can build them.
      `<ul>
            <li><a href=' URL to your profile'><img href='PATH TO THE IMAGE' /></a></li>
            <li><a href=' URL to your profile'><img href='PATH TO THE IMAGE' /></a></li>
      </ul>`
 Then after, you will style them correctly. Note that you will use `Flexbox` to layout them 😃 💡